src/install/python/python-agent-intro.mdx

```diff
@@ -0,0 +1,30 @@
+---
+componentType: default
+---
+
+import pythonagentWebTransactionChart from 'images/apm-python-agent_screenshot-crop_web-transaction-chart.png'
+
+
+Do you want to monitor the performance of your Python application? You can do this by installing our Python agent! It is a powerful tool that collects telemetry about your application and then forwards that data to New Relic. Once the data is in New Relic, you can view a wide range of UI pages that give you insights into what's happening with your software.
+
+
+    <img
+      title="Screenshot showing web transaction for Python agent"
+      alt="Screenshot showing web transaction for Python agent"
+      src={pythonagentWebTransactionChart}
+    />
+    <figcaption>
+      This is an example of the type of chart you would see in the New Relic UI if you installed the Python agent
+    </figcaption>
+
+## Start our interactive instructions
```

Comment:
If you're going to conceal the extent of the branching options that someone has to go through to get the info they need, I think it'd be helpful to add an overview of the process here.

For example, something like:

"This interactive installer asks a series of questions to get you specific installation instructions for your set up. These questions are:

1. Do you have a non-web Python app?
2. Are you using Docker?
3. What Python framework are you using?
4. What Python server are you using?"

That way people won't be surprised by the process.

Reply:
@paperclypse that's a great idea! I'll add this!